hokusai/test.yml

```diff
@@ -8,4 +8,6 @@ services:
     environment:
       - CI=true
       - CIRCLE_NODE_INDEX
+      - CDN_PRODUCTION_URL
+      - CDN_STAGING_URL
```

Comment:
I'm pretty sure using this syntax pulls in the envs from CI. I've added both of these environment variables in force's env settings in circleci's UI.